chore: FIT-1100: sync SDK

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/19970728247)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/688
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19945993396

matt-bernstein commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/19966875730)
> Successfully merged:  3 files changed, 59 insertions(+), 4 deletions(-)

matt-bernstein committed:
Merge branch 'develop' into 'fb-FIT-1100'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19966875730

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19970626794

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19970728247

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-FIT-1100'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/19970728247

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/19971919044
